### DISCUSSION TITLE
why am i fetching _rsc payload even tho everything is client component?

### DISCUSSION THREAD

**Discussion by pycraft114** [vercel/next.js#62917] [Help]
### Summary

<img width="935" alt="스크린샷 2024-03-06 오후 2 21 55" src="https://github.com/vercel/next.js/assets/24728657/99ab10ec-fd4d-4d86-bd88-b55e4f7a1229">

im using Nextjs 14 and what are all this _rsc payload that i am fetching? it doesn't event get cached while navigating pages via Link component so every time i come back to the same page loading.js is shown every time. There are no documentations for this and it is really frustrating

**Comment by icyJoseph** [comment]
If the payload is being consumed, you'll see it in the response tab.

Copy that into this tool: https://rsc-parser.vercel.app/

And there you can see what tree is being send fro the server.

**Reply by pycraft114** [reply]
@icyJoseph dynamic rendering is so slow in Nextjs 14 version,
demo: https://upground-git-test-pycraft114.vercel.app/?job=frontend
if you click the tab it will link to a new page with a new query parameter, its a really simple app but after click the tab it takes almost half a second to see the tab being highlighted, i evantually down graded to Nextjs 13 version and now its so fast, what has changed?

**Reply by pycraft114** [reply]
<img width="445" alt="스크린샷 2024-03-06 오후 8 01 40" src="https://github.com/vercel/next.js/assets/24728657/8d076053-801a-46a2-afa5-25ca3f4819dc">
it does do api fetching when new tab is clicked but the api round trip takes about 50ms on average.

when running the prod built version on local its so fast, contrary to the deployed version
